app/controllers/StripeController.scala

```diff
@@ -32,9 +33,8 @@ class StripeController(paymentServices: PaymentServices, stripeConfig: Config, o
   // THIS ENDPOINT IS USED BY BOTH THE FRONTEND AND THE MOBILE-APP
   def pay = (NoCacheAction andThen MobileSupportAction andThen ABTestAction)
     .async(BodyParsers.jsonOrMultipart(ContributionRequest.contributionForm)) { implicit request =>
-    val platform = request.platform.getOrElse("unknown")
-    info(s"A Stripe payment is being attempted with request id: ${request.id}. \n\t Request is from platform: $platform.")
-    cloudWatchMetrics.logStripePaymentAttempt(platform)
+    info(s"A Stripe payment is being attempted with request id: ${request.id}, from platform: ${request.platform}.")
+    cloudWatchMetrics.logPaymentAttempt(PaymentProvider.Stripe, request.platform)
```

Comment:
@michaelwmcnamara just realised, in this controller the `form` takes precedent when deciding what the `platform` is. Therefore the value of platform should be `form.platform.getOrElse(request.platform)`